tests/acceptance/bookmarks-test.js

```diff
@@ -49,15 +48,13 @@ module('Acceptance | bookmarks', function(hooks) {
 
   test('visiting /bookmarks', async function(assert) {
     await visit('/bookmarks');
-    await percySnapshot('default bookmarks view');
 
     assert.equal(currentURL(), '/bookmarks');
   });
 
   test('visiting /bookmarks, see empty message', async function(assert) {
     await visit('/bookmarks');
     await waitUntil(() => find('.content-area'));
-    await percySnapshot('default empty bookmarks view');
```

Comment:
Can you explain what Percy is and why you're removing stuff from this test? I don't mind changing tests to get these upgrades done (especially if the tests you're changing weren't particularly valuable in the first place), I just want to make sure we have some context documenting in GH